sim/tests/test_all.py

```diff
@@ -53,8 +61,9 @@ def test_spark_bids_skip_participant(self):
       participant_file = "skip.txt"
       with open(participant_file,"w") as f:
          f.write("01")
-      self.run_spark_bids(options=["--skip-participants","skip.txt"],correctBrainSize="865472")
-      
+      self.run_spark_bids(options=["--skip-participants", "skip.txt"],correctBrainSize="865472")
+   
+   @pytest.mark.skipif(not module_exists("hdfs"), reason="HDFS not installed")   
```

Comment:
nice workaround!

Reply:
yeah, except there's an even better way of doing it. Will fix this now